ledger/participant-integration-api/src/main/scala/platform/store/dao/JdbcLedgerDao.scala

```diff
@@ -452,21 +455,25 @@ private class JdbcLedgerDao(
 
   override def storeTransactionState(
       preparedInsert: PreparedInsert
-  )(implicit loggingContext: LoggingContext): Future[PersistenceResponse] =
+  )(implicit loggingContext: LoggingContext): Future[PersistenceResponse] = {
+    logger.info("Storing transaction state")
     dbDispatcher
       .executeSql(metrics.daml.index.db.storeTransactionDbMetrics)(
         preparedInsert.writeState(metrics)(_)
       )
       .map(_ => Ok)(servicesExecutionContext)
+  }
 
   override def storeTransactionEvents(
       preparedInsert: PreparedInsert
-  )(implicit loggingContext: LoggingContext): Future[PersistenceResponse] =
+  )(implicit loggingContext: LoggingContext): Future[PersistenceResponse] = {
+    logger.info("Storing transaction events")
```

Comment:
We can skip this log, the final one when completing is enough

Reply:
Fixed.

Reply:
It is still here :-)

Reply:
Fixed fixed.